### PR TITLE
[Spotless] Applying Google Code Format for sql files #11

### PR DESCRIPTION
**Pull request by MitchellGale**
### Description
Applies spotless for SQL directory in project. Sets checkstyle to not fail on failure for sql directory.
 
### Issues Resolved
https://github.com/opensearch-project/sql/issues/1101
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [x] New functionality has javadoc added
  - [x] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by Yury-Fridlyand — APPROVED** [review]

File: sql/src/main/java/org/opensearch/sql/sql/parser/AstAggregationBuilder.java [hunk 1]
```diff
@@ -27,6 +26,8 @@
 import org.opensearch.sql.sql.parser.context.QuerySpecification;
 
 /**
+ *
```
Comment (acarbonetto) [inline sql/src/main/java/org/opensearch/sql/sql/parser/AstAggregationBuilder.java:29]: why would spotless do this?

File: sql/src/main/java/org/opensearch/sql/sql/parser/AstExpressionBuilder.java [hunk 2]
```diff
@@ -592,43 +559,45 @@ private List<UnresolvedExpression> multiFieldRelevanceArguments(
...
+    List<UnresolvedExpression> args =
+        Arrays.asList(
+            new Literal(ctx.timestampFunction().simpleDateTimePart().getText(), DataType.STRING),
+            visitFunctionArg(ctx.timestampFunction().firstArg),
+            visitFunctionArg(ctx.timestampFunction().secondArg));
     return args;
   }
 
   /**
+   *
```
Comment (acarbonetto) [inline sql/src/main/java/org/opensearch/sql/sql/parser/AstExpressionBuilder.java:595]: ??

File: sql/src/main/java/org/opensearch/sql/sql/parser/context/QuerySpecification.java [hunk 3]
```diff
@@ -74,31 +73,36 @@ public class QuerySpecification {
   private final Set<UnresolvedExpression> aggregators = new LinkedHashSet<>();
 
   /**
+   *
```
Comment (acarbonetto) [inline sql/src/main/java/org/opensearch/sql/sql/parser/context/QuerySpecification.java:76]: ??

**Review by acarbonetto — APPROVED** [review]

File: sql/src/main/java/org/opensearch/sql/sql/parser/AstAggregationBuilder.java (hunk 1 above)
Comment (MitchellGale) [reply sql/src/main/java/org/opensearch/sql/sql/parser/AstAggregationBuilder.java:29]: Unsure...

File: sql/src/main/java/org/opensearch/sql/sql/parser/AstExpressionBuilder.java (hunk 2 above)
Comment (MitchellGale) [reply sql/src/main/java/org/opensearch/sql/sql/parser/AstExpressionBuilder.java:595]: Unsure...

**Review by matthewryanwells — APPROVED** [review]

**Review by Yury-Fridlyand — APPROVED** [review]

**Pull request merged**